### DISCUSSION TITLE
gdb/mi equivalent.

### DISCUSSION THREAD

**Discussion by epasveer** [go-delve/delve#3259] [General]
Does Delve have an equivalent to gdb's MI layer. MI is a layer for people to write GUI frontends to gdb.

**Comment by derekparker** [comment]
Hey,

You can find more information on Delves API here: https://github.com/go-delve/delve/tree/master/Documentation/api. This should have the information you need. For an example of a GUI frontend for Delve using this API see https://github.com/aarzilli/gdlv.

**Comment by epasveer** [comment]
Thanks! I'll check them out.
